src/quo/components/wallet/address_text/view.cljs

```diff
@@ -0,0 +1,30 @@
+(ns quo.components.wallet.address-text.view
+  (:require [quo.components.markdown.text :as text]
+            [quo.components.wallet.address-text.style :as style]
+            [quo.foundations.colors :as colors]
+            [quo.theme]
+            [utils.address :as utils]))
+
+(defn- colored-network-text
+  [theme network]
+  [text/text
+   {:size  :paragraph-2
+    :style {:color (colors/resolve-color network theme)}}
+   (utils/get-network-short-name network)])
+
+(defn- view-internal
+  [{:keys [networks address blur? theme format]}]
+  (let [network-text-xf (map #(colored-network-text theme %))
+        address-text    [text/text
+                         {:size   :paragraph-2
+                          ;; TODO: monospace font https://github.com/status-im/status-mobile/issues/17009
+                          :weight :monospace
+                          :style  (style/address-text format blur? theme)}
+                         (if (= format :short)
+                           (utils/get-short-wallet-address address)
+                           address)]]
+    (as-> networks $ ;; Take vector of networks
```

Comment:
are all of these comments necessary?
imo the code accompanying  ";; Take vector of networks" and ";; Add address-text hiccup at the end" is pretty self explanatory.